openawsem/functionTerms/biasTerms.py

```diff
@@ -289,14 +313,36 @@ def rg_bias_term(oa, k=1*kilocalorie_per_mole, rg0=0, atomGroup=-1, forceGroup=2
     rg.setForceGroup(forceGroup)
     return rg
 
+# Implement Wu's Rg term into OpenAWSEM. See Wu, J. Phys. Chem. B, 2018, 11115-11125.
+def IDP_term(oa, rg0=0, D=-0.5*kilocalorie_per_mole, alpha=0.001, beta=0.001, gamma=1, atomGroup=-1, forceGroup=27):
+    D = D.value_in_unit(kilojoule_per_mole)
+    nres, ca = oa.nres, oa.ca
+    if atomGroup == -1:
+        group = list(range(nres))
+    else:
+        group = atomGroup     # atomGroup = [0, 1, 10, 12]  means include residue 1, 2, 11, 13.
+    n = len(group)
+    normalization = n*n
+    rg_square = CustomBondForce(f"1.0/{normalization}*r^2")
```

Comment:
Review the units. OpenMM default units are in nanometers and the factors seem to be in Angstrom.